File: tests/python/cases/case_181_alias_shadow_cache_09.py
"""
Task case 181: alias_shadow_cache_09 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_181_alias_shadow_cache_09(input_data):
    task_name = 'alias_shadow_cache_09'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    alias = payload
    # ESCAPE: alias of payload is stored in retained cache (aliasing false-negative stress).
    RETAINED_CACHE['case_181'] = alias
    return 'ok'

File: tests/nodejs/cases/case_181_alias_shadow_cache_09.js
/**
 * Task case 181: alias_shadow_cache_09 false-positive/false-negative stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case181AliasShadowCache09(input) {
  const taskName = 'alias_shadow_cache_09';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'stress',
    stage: 'evaluation',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const alias = payload;
  // ESCAPE: alias of payload is retained (aliasing false-negative stress).
  retainedCache.set('case_181', alias);
  return 'ok';
}

module.exports = {
  case181AliasShadowCache09,
};
